webapp/src/Controller/Jury/SubmissionController.php

```diff
@@ -832,25 +821,21 @@ public function editSourceAction(Request $request, Submission $submission): Resp
 
         $formBuilder = $this->createFormBuilder($data)
             ->add('problem', EntityType::class, [
-                'class' => 'App\Entity\Problem',
+                'class' => \App\Entity\Problem::class,
```

Comment:
Shorter form.

Reply:
I search for `\App` in the code now and replaced all